test/cli/template-use/complex/out/repository/my-template/v1/src/manifest.jsonnet

```diff
@@ -41,6 +41,9 @@
       id: ConfigId("empty"),
       path: "extractor/ex-generic-v2/empty",
       rows: [],
+      metadata: {
+        "KBC.test": "value"
+      }
```

Comment:
opět v šabloně, tentokrát ve folderu `out`. (V šabloně amotné se po provedení commandu nic nezměnilo, tak tu musí být totéž jako v `in`).